File: PHP/Program/Shirt.php
<?php
require ('Clothing.php');

class Shirt extends Clothing{
 
    private $color = '';
    private $sleeve_type = '';
    
    //Konstruktor
    public function __construct( $color = '', $sleeve_type = ''){
        $this->color = $color;
        $this->sleeve_type = $sleeve_type;
      
    }
    
    //Getter setter color
    public function set_color($color){
        $this->color = $color;
    }

    public function get_color(){
        return $this->color;
    }
    
    //Getter setter sleeve type
    public function set_sleeve_type($sleeve_type){
        $this->sleeve_type = $sleeve_type;
    }

    public function get_sleeve_type(){
        return $this->sleeve_type;
    }

  

}

?>

File: Pyhton/Program/Shirt.py
from Clothing import Clothing

class Shirt(Clothing):
    def __init__(self):
        super().__init__()
        self.color = ""
        self.sleeve_type = ""

    # Getter dan setter untuk color
    def getcolor(self):
        return self.color

    def setcolor(self, color):
        self.color = color

    # Getter dan setter untuk sleeve_type
    def getsleeve_type(self):
        return self.sleeve_type

    def setsleeve_type(self, sleeve_type):
        self.sleeve_type = sleeve_type
